prac_06/guitar_test.py

```diff
@@ -0,0 +1,26 @@
+"""
+CP1404 Practical 06 - Test Guitar class
+Time estimated: 20 minutes
+Time taken: 19 minutes
+"""
+
+from prac_06.guitar import Guitar
+
+
+def run_tests():
+    """Test for Guitar class."""
+    name = "Gibson L-5 CES"
+    year = 1922
+    cost = 16035.40
+
+    guitar = Guitar(name, year, cost)
```

Comment:
Maybe you did not need to put the guitar info in variables, you could of put them into the class as arguments. Like for 'another_guitar' in the line below.